numpy/core/fromnumeric.py

```diff
@@ -70,11 +70,21 @@ def take(a, indices, axis=None, out=None, mode='raise'):
     using arrays); however, it can be easier to use if you need elements
     along a given axis.
```

Comment:
I'm also a little surprised this doesn't explicitly say somthing like " this is equivalent to `a[:,:,:,indices,:,:,:]` with indices placed in the axis `axis`".